.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Should we commit it? 

IHMO it is just the VSCODE file created for who uses this IDEA then it is not pertinent to the project and people are able to use other IDEAs. I don't think that it should be committed.

Reply:
@camilamacedo86 I also wasn't using vscode but I was advised by @davidffrench that the team all use it so I installed it and I wrote the steps to develop/debug locally based on the use of that. In that case it could be helpful to have the configuration already in the project files in order to help someone new to the project, that way they don't need to do the steps to generate it.

Reply:
@laurafitzgerald it is fine. However, this file will be generated automatically when open the project in the IDEA VSCode. So, it is not required to commit it. I understand that we should not commit the IDEA config files and they should be added in the .gitignore. 

PS.: I do not use VSCode. I normally use the Intellij ideas :-)

Reply:
disagree automatic launch.json is very different
```
{
    // Use IntelliSense to learn about possible attributes.
    // Hover to view descriptions of existing attributes.
    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Launch",
            "type": "go",
            "request": "launch",
            "mode": "auto",
            "program": "${fileDirname}",
            "env": {},
            "args": []
        }
    ]
}
```

Reply:
@austincunningham are you saying we should include it?

Reply:
@laurafitzgerald he is.

Reply:
:+1:

Reply:
IHMO it can be added in the readme as config for who would like to use VSCode IDEA since it has customizations and it is not the default file generated. **REASON: It is not a good practice to add local config files in the projects. It is specific for who would like to use VS CODE and it and is not part of the project at all.** 

Also, note that:

<img width="876" alt="screenshot 2019-02-12 at 09 31 16" src="https://user-images.githubusercontent.com/7708031/52625473-fc130880-2ea8-11e9-8984-5381fd59fffe.png">

Following some links.  

- https://softwareengineering.stackexchange.com/questions/202051/what-should-i-include-in-my-repositiory-from-ide-projects

> The repo should be clean and tidy and only have those things which are relevant to the project not your environment.

- https://hackernoon.com/exclude-files-from-git-without-committing-changes-to-gitignore-986fa712e78d

> Maybe there are some files, however, that you don’t want to commit to version control because:
> 
> Local settings, scripts or other configuration files that are specific to your local development environment.

- https://gist.github.com/pandeiro/1552496#dont

<img width="957" alt="screenshot 2019-02-12 at 09 18 20" src="https://user-images.githubusercontent.com/7708031/52624745-557a3800-2ea7-11e9-9d85-28ea23162fe2.png">

- https://datree.io/blog/top-10-github-best-practices/

> We strongly recommend against committing your local config files to version control.

Reply:
Tend to agree with @camilamacedo86 as it's IDE-specific. An alternative to including the file with the repo would be to document it in the README

Reply:
@camilamacedo86 @damienomurchu yes the more I think about it the more I think we shouldn't include it.  I will make the required updates to the README to provide guidance on how to get the config set up locally.

Reply:
Hi @laurafitzgerald you made a great job here. I am approving it since IHMO all here shows perfect now !!!.